### PR TITLE
Hot fix dev

### PR DESCRIPTION
**Pull request by NorahAba**
add page background function and UI
solve cell background problem
add font color to the output cell

### CODE REVIEW

File: src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/main.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ define([
     css_initial(
       "../../nbextensions/accessibility_toolbar/font_style/predefined_styles.css"
     );
-    css_initial("../../nbextensions/accessibility_toolbar/theme_style/themes.css");
+    // css_initial("../../nbextensions/accessibility_toolbar/theme_style/themes.css");
```
Comment (joshuazeltser) [inline src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/main.js:36]: Can you uncomment this for dev

File: src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js [hunk 2]
```diff
@@ -7,11 +7,18 @@ define(["base/js/namespace", "jquery", "./spectrum"], function(
 
   //=============
```
Comment (joshuazeltser) [inline src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js:8]: Please remove all commented out code from the file

**Review by joshuazeltser — CHANGES_REQUESTED** [review]

File: src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js (hunk 2 above)
Comment (NorahAba) [reply src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js:8]: Sure I will do :)
If the code works for you guys, I will solve the conflict

Comment (joshuazeltser) [reply src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js:8]: Once you've solved conflicts ill double check all works then merge

File: src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js [hunk 3]
```diff
@@ -222,42 +323,207 @@ define(["base/js/namespace", "jquery", "./spectrum"], function(
...
+      //   1
+      // );
+    }
+    rule = style_Sheet.cssRules;
+  };
+  //end set color
+
+  //=======================================================
+  //=======================================================
+  Color_control.prototype.set_markdown_color = function() {
```
Comment (joshuazeltser) [inline src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js:363]: There is no need for this function as set-color changes markdowns as well

File: src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js [hunk 4]
```diff
@@ -222,42 +323,207 @@ define(["base/js/namespace", "jquery", "./spectrum"], function(
...
+      console.log("rule insert");
+    }
+    rule2 = style_Sheet2.cssRules;
+  };
+  //end  set page background color
+
+  //==============================
+
+  //==============================
+  Color_control.prototype.page_font_set_color = function() {
```
Comment (joshuazeltser) [inline src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/color_control.js:474]: same as above

**Review by joshuazeltser — CHANGES_REQUESTED** [review]

**Comment by joshuazeltser** [comment]
I have merged your branch into mine as i needed your changes for my things so no need to solve the conflicts now. Also, i made some small changes based on some of my comments above

**Pull request closed without merging**